### ISSUE TITLE
Pre-work Checklist: Developer: Jeristella

### ISSUE THREAD

**Issue by jeristella** [hackforla/website#3908]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [x] #3920
- [x] Share your GitHub handle and Gmail address in the hfla-site-onboarding slack channel so you can be added to the Google Drive (alternative to go to meeting and direct message a merge team member at the [meeting](https://github.com/hackforla/website/wiki/Meetings-and-Agendas
))
  - [x] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [x] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [ ] Attend weekly team meetings:
  - [ ] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [ ] (Optional) Office Hours, Thursdays 7-8pm PST
  - [ ] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note: There are no meetings on the 1st-7th of every month.
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [ ] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by ExperimentsInHonesty** [comment]
@jeristella How are you doing on setting up your development environment? It looks like you have not yet assigned yourself to a good first issue. If you need help please reach out in the hfla-site slack channel or come to a team meeting.

**Comment by ExperimentsInHonesty** [comment]
I see that you added product.  If you are interested in a product management role on the team, please talk to me at the next sunday team meeting.  If that time doesn't work for you, please move this issue to the questions column and provide some times that do work for you.

**Comment by jeristella** [comment]
@bonniewolfe Hi Bonnie! I will try to complete setting up my development environment by this week and assign myself an issue. 

I am also interested in product management. I will get in touch with you at the next Sunday meeting.

**Comment by jeristella** [comment]
How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?

1 month, spaced  throughout ~15 hours

**Comment by jeristella** [comment]
1. Progress: In the process of choosing a second issue to work on. 
2. Blockers: nothing at the moment.
3. Just finished final exams, so have more availability to work on issue. 11 am to 5 pm March 27  to March 31. 
4. ETA: April 1st 10pm

**Comment by ExperimentsInHonesty** [comment]
@jeristella I noticed that you have not picked up a second issue.  It's possible when you looked for one, there was not one available.  

There is 1 available now [Good front end](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22+label%3A%22role%3A+back+end%2Fdevops%22+label%3A%22complexity%3A+good+second+issue%22#column-7198257).  

If you don't find one when you look again, go ahead and do another [good first issue](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22good+first+issue%22+label%3A%22role%3A+front+end%22+label%3A%22role%3A+back+end%2Fdevops%22#column-7198257).  They are basically the same type of issue, we just need people to do two total so that we can make sure you dev environment is setup and you know how to do branching effectively.

**Comment by jeristella** [comment]
Hi Bonnie! Sounds good, I've assigned myself. I expect to complete it by the end of the week.

**Comment by jdingeman** [comment]
Hi @jeristella - please provide us with an update on your overall status. Thanks!

**Comment by jeristella** [comment]
Hi!

I think I'm overall done with the prework checklist. Shall I go ahead and close the issue? I will also start to look for new smaller issues to work on. Spring quarter has started for us at uni so will be a bit busy with assignments and exams. I should be able to work on some issues on weekends.

**Comment by roslynwythe** [comment]
Hi @jeristella please leave this issue open as you progress through the sequence of issues, and please leave a weekly progress report.  If there has been no comment activity in an issue for 2 weeks, the issue is flagged for review by dev leads.  Please follow the instructions in https://github.com/hackforla/website/issues/3908#issuecomment-1482360818 for leaving an update and removing the '2 week inactive' label.  Thank you!

**Comment by roslynwythe** [comment]
Hi @jeristella this issue has been flagged for review by dev leads because there have been no updates in the past several weeks.  If you wish to continue with HfLA, please provide updates at least every two weeks.  That way we know that you are still an active member and we can provide support based on your reported progress.

**Comment by jeristella** [comment]
Hello, 

Sorry for the late response! I haven't had the time to take on new issues. School has been very busy and stressful. Because of this and other commitments, I unfortunately would need to take a break working for the website team. 

Kind regards,

Jeristella

**Comment by ExperimentsInHonesty** [comment]
Hi @jeristella I hope all is well with you and school is going well.  I see that you did two good first issues and then did not pick up a small issue. If you want to come back to the team and pick up where you left off, then please go ahead and self assign a [small issue from the prioritized backlog](https://github.com/hackforla/website/projects/7?card_filter_query=-label%3A%22role%3A+dev+leads%22++label%3A%22role%3A+front+end%22+label%3A%22complexity%3A+small%22#column-7198257).  If you find you have lost your rights to self assign an issue, please message us in the hfla-site Slack channel.

**Comment by HackforLABot** [comment]
## Activity Log: jeristella
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- jeristella opened: https://github.com/hackforla/website/issues/3908 at 01/31/2023, 08:55 PM PST
- jeristella assigned: https://github.com/hackforla/website/issues/3908 at 01/31/2023, 08:58 PM PST
- jeristella closed as not planned: https://github.com/hackforla/website/issues/3908#event-8406580148 at 01/31/2023, 09:37 PM PST
- jeristella reopened: https://github.com/hackforla/website/issues/3908#event-8406580148 at 01/31/2023, 09:45 PM PST
- jeristella commented on: https://github.com/hackforla/website/issues/3908#issuecomment-1445471204 at 02/26/2023, 01:24 PM PST
- jeristella assigned: https://github.com/hackforla/website/issues/4043 at 02/27/2023, 11:44 PM PST
- jeristella opened: https://github.com/hackforla/website/pull/4097 at 03/02/2023, 10:07 PM PST
- jeristella commented on: https://github.com/hackforla/website/issues/4043#issuecomment-1453035929 at 03/02/2023, 10:16 PM PST
- jeristella commented on: https://github.com/hackforla/website/issues/3908#issuecomment-1453038380 at 03/02/2023, 10:20 PM PST
- jeristella closed: https://github.com/hackforla/website/pull/4097#event-8657382164 at 03/03/2023, 12:29 AM PST
- jeristella reopened: https://github.com/hackforla/website/pull/4097#event-8657382164 at 03/03/2023, 01:05 AM PST
- jeristella opened: https://github.com/hackforla/website/pull/4099 at 03/03/2023, 01:51 AM PST
- jeristella PR was closed: https://github.com/hackforla/website/pull/4097#event-8668702363 at 03/05/2023, 01:16 PM PST
- jeristella PR was merged: https://github.com/hackforla/website/pull/4099#event-8680279722 at 03/06/2023, 04:32 PM PST
- jeristella issue closed by PR 4099: https://github.com/hackforla/website/issues/4043#event-8680279864 at 03/06/2023, 04:32 PM PST
- jeristella commented on: https://github.com/hackforla/website/pull/4099#issuecomment-1462912369 at 03/09/2023, 02:25 PM PST
- jeristella commented on: https://github.com/hackforla/website/issues/3908#issuecomment-1483421407 at 03/24/2023, 02:24 PM PDT
- jeristella assigned: https://github.com/hackforla/website/issues/4239 at 03/27/2023, 12:18 PM PDT
- jeristella commented on: https://github.com/hackforla/website/issues/4239#issuecomment-1485735457 at 03/27/2023, 12:19 PM PDT
- jeristella commented on: https://github.com/hackforla/website/issues/3908#issuecomment-1485736703 at 03/27/2023, 12:20 PM PDT
- jeristella opened: https://github.com/hackforla/website/pull/4309 at 03/28/2023, 08:13 PM PDT
- jeristella commented on: https://github.com/hackforla/website/pull/4309#issuecomment-1490972717 at 03/30/2023, 02:15 PM PDT
- jeristella PR was merged: https://github.com/hackforla/website/pull/4309#event-8980569459 at 04/11/2023, 08:50 PM PDT
- jeristella issue closed by PR 4309: https://github.com/hackforla/website/issues/4239#event-8980569586 at 04/11/2023, 08:50 PM PDT
- jeristella commented on: https://github.com/hackforla/website/issues/3908#issuecomment-1512306479 at 04/17/2023, 06:24 PM PDT
- jeristella commented on: https://github.com/hackforla/website/issues/3908#issuecomment-1550451529 at 05/16/2023, 03:43 PM PDT
- jeristella closed as completed: https://github.com/hackforla/website/issues/3908#event-9382632878 at 05/30/2023, 01:20 PM PDT
